src/main/java/uk/gov/hmcts/reform/sscs/validators/SscsCaseValidator.java

```diff
@@ -216,7 +216,10 @@ private void checkAppointee(Appellant appellant, Map<String, Object> ocrCaseData
     }
 
     private void checkRepresentative(Appeal appeal, Map<String, Object> ocrCaseData, Map<String, Object> caseData) {
-        if (appeal.getRep() != null && appeal.getRep().getHasRepresentative().equals(YES_LITERAL)) {
+        if (appeal.getRep() == null || StringUtils.isBlank(appeal.getRep().getHasRepresentative())) {
```

Comment:
Do we just want to display warning or error? error will not create the bulk-scan case as we have auto case creation
Also this validation is called before transforming the request and case validation thought CCD UI

Reply:
The A/C on the ticket says error.
```
If I have created an incomplete case via the UI

Using the "Create new" button and selecting "Incomplete application rec'd"
And then populate all necessary information, but leave the “Has representative” unchecked
And then proceed to use the “Validate appeal” event
Then I show see the error message below:
“The “Has representative” field is not selected, please select an option to proceed”
```

Reply:
The ticket says SYA and Bulk scan cases will always have a Rep
```
Technical
The area of the code that throws the exception is –
https://github.com/hmcts/sscs-bulk-scan/blob/3628ce7b52555aaa676f037e0952a21b205334be/src/main/java/uk/gov/hmcts/reform/sscs/validators/SscsCaseValidator.java#L242

Existing bulk scan
For bulk scanning cases, if the representative is seen in the OCR data, then the affected field is automatically populated, so it wouldn't trigger the error seen above - https://github.com/hmcts/sscs-bulk-scan/blob/eb2fea2241f7639bc6c5cd559f643602875309fa/src/main/java/uk/gov/hmcts/reform/sscs/transformers/SscsCaseTransformer.java#L286

Existing SYA
For SYA cases, if the representative is seen, then the affected field is also populated -
https://github.com/hmcts/sscs-tribunals-case-api/blob/baa019cec37d7c660513c2521558ab0cb51b0ea8/src/main/java/uk/gov/hmcts/reform/sscs/transform/deserialize/SubmitYourAppealToCcdCaseDataDeserializer.java#L657
```